integration-tests/src/test/groovy/com/okta/sdk/tests/it/spec/GroupRuleIT.groovy

```diff
@@ -94,7 +94,7 @@ class GroupRuleIT extends ITSupport {
         rule.activate()
 
         // Triggering the updated rule will remove the user from group
-        assertUserInGroup(user, group, 20, 1000, false)
+        assertUserInGroup(user, group, 120, 1000, false)
```

Comment:
flaky test, depending on server timing, using mock responses will help with this in the near future